specs/alignments.yaml

```diff
@@ -0,0 +1,52 @@
+url: https://github.com/airr-community/airr-formats/blob/master/specs/alignments.yaml
+version: 0.1.0
+name: rearrangement
+type: object
+description: AIRR schema for Ig/TCR rearrangements
+fields:
+  -
+    name: sequence_id
+    type: string
+    doc: Read/sequence identifier
+    mandatory: True
+  -
+    name: sequence
+    type: string
+    doc: Nucleotide sequence (e.g., the "read" sequence; revcomp'd if
+         necessary)
+    mandatory: True
+  -
+    name: segment
+    type: string
+    doc: The gene segment for this alignment (e.g., V, D, J, C)
+    mandatory: True
+  -
+    name: rev_comp
+    type: boolean
+    doc: Sequence is reverse complemented
+    mandatory: True
+  -
+    name: segment_call
+    type: string
+    doc: Allele assignment
+    mandatory: True
+  -
+    name: segment_score
+    type: float
+    doc: Alignment score
+    mandatory: True
+  -
+    name: segment_cigar
+    type: string
+    doc: Alignment CIGAR string
+    mandatory: True
+  -
+    name: segment_evalue
```

Comment:
Probably? I know we'll need `evalue` for IgBLAST output and I suspect we'll need `probability` and `loglikelihood` for aligners like partis. @matsen and @psathyrella?